packages/governance/src/committeeRegistrar.js

```diff
@@ -0,0 +1,84 @@
+// @ts-check
+
+import { Far } from '@agoric/marshal';
+import { makeNotifierKit } from '@agoric/notifier';
+import { E } from '@agoric/eventual-send';
+import { makeStore } from '@agoric/store';
+import { allComparable } from '@agoric/same-structure';
+
+// Each CommitteeRegistrar represents a particular set of voters. The number of
+// voters is visible in the terms.
+const start = zcf => {
+  // Question => { voter, publicFacet }
+  const allQuestions = makeStore('Question');
+  const { notifier, updater } = makeNotifierKit();
+  const invitations = [];
+
+  const getOpenQuestions = async () => {
+    const isOpenPQuestions = allQuestions.keys().map(key => {
+      const { publicFacet } = allQuestions.get(key);
+      return [E(publicFacet).isOpen(), key];
+    });
+    const isOpenQuestions = await allComparable(harden(isOpenPQuestions));
+    return isOpenQuestions
+      .filter(([open, _key]) => open)
+      .map(([_open, key]) => key);
+  };
+
+  const makeCommitteeVoterInvitation = index => {
+    const handler = voterSeat => {
+      return Far(`voter${index}`, {
+        castBallot: ballot => {
+          const { voter } = allQuestions.get(ballot.question);
+          return E(voter).submitVote(voterSeat, ballot);
+        },
+      });
+    };
+
+    return zcf.makeInvitation(handler, `Voter${index}`);
+  };
+
+  const { committeeName, committeeSize } = zcf.getTerms();
+  for (let i = 0; i < committeeSize; i += 1) {
+    invitations[i] = makeCommitteeVoterInvitation(i);
+  }
+
+  /** @type {AddQuestion} */
+  const addQuestion = async (voteCounter, questionDetailsShort) => {
+    const questionDetails = {
+      ...questionDetailsShort,
+      registrar: zcf.getInstance(),
+    };
+    // facets of the ballot counter. Suppress creatorInvitation and adminFaceet.
+    const { creatorFacet, publicFacet, instance } = await E(
+      zcf.getZoeService(),
+    ).startInstance(voteCounter, {}, questionDetails);
```

Comment:
Feel free to take it or leave it:

```suggestion
    ).startInstance(voteCounter, undefined, questionDetails);
```

`{}` should work now that we don't treat empty objects as presences, but I think `undefined` is clearer

Reply:
I prefer the curly braces. It's easier for me to recognize those as empty terms.